lib/compiler/src/compile.erl

```diff
@@ -773,6 +775,16 @@ select_list_passes_1([P|Ps], Opts, Acc) ->
 select_list_passes_1([], _, Acc) ->
     {not_done,reverse(Acc)}.
 
+make_ssa_check_pass(PassFlag) ->
+    F = fun (Code, St) ->
+                case beam_ssa_check:module(PassFlag, Code) of
```

Comment:
All other `module/2` functions take the code as the first argument and the option list as the second argument. I suggest that you adhere to that convention.

```suggestion
                Options = [{pass_flag, PassFlag} | St#compile.options], 
                case beam_ssa_check:module(Code, Options) of
```